contrib/test_jenkins.sh

```diff
@@ -306,6 +306,26 @@ get_rdma_device_ip_addr() {
 	echo $ipaddr
 }
 
+get_non_rdma_ip_addr() {
+    if ! which ibdev2netdev >&/dev/null
```

Comment:
pls use tabs for indentation

Reply:
I updated. 
sorry, didn't notice before that the whole function didn't have the correct indentation